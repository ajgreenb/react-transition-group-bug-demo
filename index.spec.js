var React = require('react');
var shallow = require('enzyme').shallow;
var Demo = require('./');

test('Demo', function () {
  var el = React.createElement(Demo);
  expect(shallow(el)).toMatchSnapshot();
});
