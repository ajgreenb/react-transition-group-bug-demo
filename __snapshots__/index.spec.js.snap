// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Demo 1`] = `
<CSSTransitionGroup
  transitionAppear={false}
  transitionEnter={true}
  transitionEnterTimeout={300}
  transitionLeave={true}
  transitionLeaveTimeout={300}
  transitionName="demo"
>
  This is a test.
</CSSTransitionGroup>
`;
